index.js

```diff
@@ -199,13 +199,13 @@ var Base = Class.extend({
     }
 
     var pkSql = '';
-    if (primaryKeyColumns.length > 1) {
-      pkSql = this._handleMultiPrimaryKeys(primaryKeyColumns);
-
-    } else {
-
-      primaryKeyColumns[0] = primaryKeyColumns[0].name;
-      columnDefOptions.emitPrimaryKey = true;
+    if (primaryKeyColumns.length) {
```

Comment:
```javascript
const array = [];
if (array.length > 0) {
 // do something
}
```
Is fundamentally the same as:
```javascript
const array = [];
if (array.length) {
 // do something
}
```

I do understand that the former is more readable, so if you'd very much like me to change it I will. However the latter is more succinct.